packages/pieces/community/invoiceninja/src/lib/actions/create-invoice.ts

```diff
@@ -0,0 +1,212 @@
+import {
+  createAction,
+  Property,
+  Validators,
+} from '@activepieces/pieces-framework';
+
+import { invoiceninjaAuth } from '../..';
+
+export const createInvoice = createAction({
+  auth: invoiceninjaAuth,
+  name: 'create_invoice',
+  displayName: 'Create Invoice',
+  description: 'Creates an invoice in Invoice Ninja for billing purposes.',
+  
+  props: {
+    client_id: Property.LongText({
+      displayName: 'Client ID (alphanumeric)',
+      description: 'Client ID from Invoice Ninja',
+      required: true,
+    }),
+    purchase_order_no: Property.LongText({
+      displayName: 'Purchase Order Number (alphanumeric)',
+      description: 'Descriptive text or arbitrary number (optional)',
+      required: false,
+    }),
+    discount: Property.LongText({
```

Comment:
Can you please change the discount type to a number? As API docs says it's number field.